sdk/maps/maps-route-rest/swagger/README.md

```diff
@@ -8,6 +8,8 @@ The configuration is following the [RLC quick start guide](https://github.com/Az
 For the configuration property, please refer to [Index of AutoRestFlag](https://github.com/Azure/autorest/blob/main/docs/generate/flags.md).
 
 ```yaml
+flavor: azure
```

Comment:
I notice this is still generated from Swagger, @xirzec I wonder what is our plan for data-plane to adopt TypeSpec.